core/src/allocation_init.jl

```diff
@@ -460,27 +243,29 @@ Only finite capacities get a constraint.
 The constraint indices are (edge_source_id, edge_dst_id).
 
 Constraint:
-flow over edge <= edge capacity
+flow over edge ≤ edge capacity
 """
 function add_constraints_capacity!(
     problem::JuMP.Model,
     capacity::JuMP.Containers.SparseAxisArray{Float64, 2, Tuple{NodeID, NodeID}},
     p::Parameters,
-    allocation_network_id::Int32,
+    subnetwork_id::Int32,
 )::Nothing
-    (; graph) = p
-    main_network_source_edges = get_main_network_connections(p, allocation_network_id)
+    main_network_source_edges = get_main_network_connections(p, subnetwork_id)
     F = problem[:F]
+
+    # Find the edges within the subnetwork with finite capacity
     edge_ids_finite_capacity = Tuple{NodeID, NodeID}[]
     for (edge, c) in capacity.data
         if !isinf(c) && edge ∉ main_network_source_edges
             push!(edge_ids_finite_capacity, edge)
         end
     end
+
     problem[:capacity] = JuMP.@constraint(
         problem,
         [edge = edge_ids_finite_capacity],
-        F[edge] <= capacity[edge...],
+        F[edge] ≤ capacity[edge...],
```

Comment:
Do you use a font with coding ligature support? To me a `<` followed by a `=` already looks like one symbol. I use Cascadia Code, see https://github.com/microsoft/cascadia-code?tab=readme-ov-file#font-features.

If you use that perhaps we can just keep the ASCII version?